source/includes/_jobbrandings.md

```diff
@@ -0,0 +1,187 @@
+
+## Get a Single Job Branding
+To get a single job branding by it’s Id you have to use following request:
+
+### HTTP Request
+
+> This request returns JSON structured like this:
+
+```json
+{
+  "href": "https://api.1brd.com/v1/companies/00000000-0000-0000-0000-000000000000/job_brandings/00000000-0000-0000-0000-000000000001",
+  "id": "00000000-0000-0000-0000-000000000001",
+  "company": {
+    "id": "00000000-0000-0000-0000-000000000000",
+    "href": "https://api.1brd.com/v1/companies/00000000-0000-0000-0000-000000000000"
+  },
+  "name": "Branding One",
+  "isDefault": false,
+  "shareLogo": {
+    "id": "00000000-0000-0000-0000-0000000000002",
+    "href": "https://localhost/share-logo-image.jpeg"
+  },
+  "linkedInShareLogo": {
+    "id": "00000000-0000-0000-0000-000000000002",
+    "href": "https://localhost/linkedin-share-logo-image.jpeg"
+  },
+  "jobHeader": {
+    "id": "00000000-0000-0000-0000-000000000003",
+    "href": "https://localhost/job-header-image.jpeg"
+  },
+  "shareDescription": "This is my share description.",
+  "createdAt": "2018-10-09T10:03:39Z",
+  "updatedAt": "2018-10-09T10:04:38Z"
+}
+```
+
+`GET /v1/companies/{companyId}/job_brandings/{jobBrandingId}`
+
+### Response Fields
+
+| Property              | Required | Description                                                                                           |
+|:----------------------|:---------|:------------------------------------------------------------------------------------------------------|
+| href                  | Yes      | The link to this job branding.
+| id                    | Yes      | The id of this job branding.
+| company               | Yes      | The company this job branding is associated with.
+| name                  | No       | The name of this job branding. Absent, if this is the default branding.
+| isDefault             | Yes      | Whether this job branding is the default one or not.
+| shareLogo             | Yes      | The share logo that will be used for this job branding.
+| linkedInShareLogo     | Yes      | The share logo that will be used for this job branding on LinkedIn.
+| jobHeader             | Yes      | The job header image that will be used for this job branding.
+| shareDescription      | Yes      | The share description that will be used for this job branding.
+| createdAt             | No       | The date time when this job branding got created. Absent, if this is the default branding.
+| updatedAt             | No       | The date time when this job branding got updated. Absent, if this job branding has never been updated.
+
+## Create a Job Branding
+
+To create a new job branding for your company use following request:
+
+### HTTP Request
+
+> This is an example request body
+
+```json
+{
+  "name": "Branding One",
+  "jobHeader": {
+    "id": "00000000-0000-0000-0000-000000000003"
+  },
+  "shareLogo": {
+    "id": "00000000-0000-0000-0000-000000000002"
```

Comment:
Is there already documentation on how to upload files?

Reply:
actually not...i will add that as well